specs/proposals.md

```diff
@@ -47,43 +49,31 @@ output, and the appropriate timestamp, to the [L2 Output Commitment](#l2-output-
-  latest_block: ExecutionPayload  # includes block hash
-  history_accumulator_root: Bytes32  # Not functional yet
-  extension: Bytes32
+```pseudocode
+keccak256(version_byte . state_root . withdrawal_storage_root . latest_block)
 ```
 
-The `state_root` is the Merkle-Patricia-Trie ([MPT][g-mpt]) root of all execution-layer accounts,
-also found in `latest_block.state_root`: this field is frequently used and thus elevated closer to the L2 output root,
-as opposed to retrieving it from the pre-image of the block in `latest_block`,
-reducing the merkle proof depth and thus the cost of usage.
+where the concatenated variables are:
 
-The `withdrawal_storage_root` elevates the Merkle-Patricia-Trie ([MPT][g-mpt]) root of L2 Withdrawal contract storage.
-Instead of a MPT proof to the Withdrawal contract account in the account trie,
-one can directly access the MPT storage trie root, thus reducing the verification cost of withdrawals on L1.
+1. The `version_byte` (`bytes32`) a simple version string which increments anytime the construction of the output root
+   is changed.
 
-The `latest_block` is an execution-layer block of L2, represented as the [`ExecutionPayload`][ExecutionPayload] SSZ type
-defined in L1. There may be multiple blocks per L2 output root, only the latest is presented.
+1. The `state_root` (`bytes32`) is the Merkle-Patricia-Trie ([MPT][g-mpt]) root of all execution-layer accounts,
+   also found in `latest_block.state_root`: this field is frequently used and thus elevated closer to the L2 output
+   root, as opposed to retrieving it from the pre-image of the block in `latest_block`, reducing the merkle proof depth
+   and thus the cost of usage.
 
-[ExecutionPayload]: https://github.com/ethereum/consensus-specs/blob/dev/specs/bellatrix/beacon-chain.md#executionpayload
+1. The `withdrawal_storage_root` (`bytes32`) elevates the Merkle-Patricia-Trie ([MPT][g-mpt]) root of L2 Withdrawal
+   contract storage. Instead of a MPT proof to the Withdrawal contract account in the account trie, one can directly
+access the MPT storage trie root, thus reducing the verification cost of withdrawals on L1.
 
-The `history_accumulator_root` is a reserved field, elevating a storage variable of the L2 chain that maintains
-the [SSZ] merkle root of an append-only `List[Bytes32, MAX_ITEM_COUNT]` (`keccak256` [SSZ] hash-tree-root),
-where each item is defined as `keccak256(l2_block_hash ++ l2_state_root)`, one per block of the L2 chain.
-While reserved, a zeroed `Bytes32` is used instead.
-This is a work-in-progress, see [issue 181](https://github.com/ethereum-optimism/optimistic-specs/issues/181).
-`MAX_ITEM_COUNT` and/or other parameters will be defined in the withdrawals milestone.
+1. The `latest_block` (`bytes32`) is an execution-layer block of L2, represented as the
+   [`ExecutionPayload`][executionpayload] SSZ type defined in L1. There may be multiple blocks per L2 output root, only
+   the latest is presented.
```

Comment:
"execution-layer block" sounds weird to me, because we don't really make this kind of distinction anywhere (distinction between what even?) and it will lead readers to wonder if there's something to it. Can we just say "L2 block"?